docs/compare.md

```diff
@@ -22,4 +22,18 @@ To compare two load test reports, run:
 go run ./cmd/ltctl report compare base.out new.out --output=results.txt --graph
 ```
 
-The results.txt will be a markdown formatted table comparing the average and p99 times of the store and API metrics. Additionally, a `--graph` parameter can also be passed which can be used to generate graphs comparing different metrics like CPU, Memory etc. This also requires the `gnuplot` command to be installed on the system for it to plot graphs.
\ No newline at end of file
+The results.txt will be a markdown formatted table comparing the average and p99 times of the store and API metrics. Additionally, a `--graph` parameter can also be passed which can be used to generate graphs comparing different metrics like CPU, Memory etc. This also requires the `gnuplot` command to be installed on the system for it to plot graphs.
+
+## Best practices while comparing load-tests
+
+- Always use the same cluster setup to compare different tests.
+- Calculate the number of users required to reach an average time of at least 10ms. And then compare the tests using that number of users. Anything less than that means the DB is not being stressed well enough and can introduce a lot of noise in the results.
+- While starting another test, it is recommended to reset the DB and initialize it again to start with the exact same state as the previous test. Follow these steps to accomplish that.
+
+On the server, run:
+- `/opt/mattermost$ echo YES | ./bin/mattermost reset`
+- `/opt/mattermost$ ./bin/mattermost user create --email sysadmin@sample.mattermost.com --username sysadmin --password Sys@dmin-sample1 --system_admin`
+- `sudo service mattermost restart`
```

Comment:
Did we ever have a ticket to make this work through `ltctl`?

Reply:
No, I don't think so. Created https://mattermost.atlassian.net/browse/MM-26443.